webapp/src/shared/components/ChatBubble/ChatBubble.tsx

```diff
@@ -23,7 +23,9 @@ export default function ChatBubble({
   return (
     <div className={`chat-bubble chat-bubble-${variant}`}>
       {avatar && <SmallAvatar {...avatar} />}
-      <div className={'chat-bubble-text-container'}>
+      <div
+        className={`chat-bubble-text-container chat-bubble-text-container-${variant}`}
+      >
         <div className={'chat-bubble-text-author'}>
```

Comment:
I think we should add variants to this chat bubble such as 'is-consecutive' so that we can maybe not show the author nickname when there are a series of bubbles that are all from the same author, or not show the avatar if it's not the last one in a sequence of such bubbles.

Reply:
Thanks! I like this :+1:

I've made this change. Please let me know what we think or if there is any way to improve it.

Reply:
I will look at it tomorrow on my workstation (I'm on my phone more). Thank you!

Reply:
What about this
```
diff --git a/webapp/src/pages/ChatRoom/Messages.tsx b/webapp/src/pages/ChatRoom/Messages.tsx
index e478db1..c6e6c47 100644
--- a/webapp/src/pages/ChatRoom/Messages.tsx
+++ b/webapp/src/pages/ChatRoom/Messages.tsx
@@ -45,6 +45,10 @@ function Messages() {
         const isConsecutive =
           index !== messages.length - 1 &&
           messages[index].user.id === messages[index + 1].user.id;
+        const isFirst =
+          index === 0 ||
+          messages[index].user.id !== messages[index - 1].user.id;
+        console.log(isFirst);
         return (
           <li key={message.id} className="messages-list-item">
             <ChatBubble
@@ -61,6 +65,7 @@ function Messages() {
                   : WILDCARD_AVATAR_URL,
               }}
               isConsecutive={isConsecutive}
+              isFirst={isFirst}
             />
           </li>
         );
diff --git a/webapp/src/shared/components/ChatBubble/ChatBubble.tsx b/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
index 407807c..d0b01a3 100644
--- a/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
+++ b/webapp/src/shared/components/ChatBubble/ChatBubble.tsx
@@ -13,6 +13,7 @@ type ChatBubbleProps = {
   text: string;
   name?: string;
   isConsecutive?: boolean;
+  isFirst?: boolean;
 };
 
 export default function ChatBubble({
@@ -21,6 +22,7 @@ export default function ChatBubble({
   text,
   name,
   isConsecutive = false,
+  isFirst = true,
 }: ChatBubbleProps) {
   return (
     <div className={`chat-bubble chat-bubble-${variant}`}>
@@ -31,7 +33,7 @@ export default function ChatBubble({
         className={`chat-bubble-text-container chat-bubble-text-container-${variant}`}
       >
         <div className={'chat-bubble-text-author'}>
-          {name && !isConsecutive && (
+          {name && isFirst && (
             <Text
               variant={TextVariant.CAPTION}
               color={TextColor.LIGHT}
```

?

Reply:
LGTM :+1: